homeassistant/components/ovo_energy/config_flow.py

```diff
@@ -84,6 +96,8 @@ async def async_step_reauth(self, user_input):
             else:
                 if authenticated:
                     entry = await self.async_set_unique_id(self.username)
+                    if entry is None:
+                        return self.async_abort(reason="unknown")
```

Comment:
Is this expected to happen in real life? Maybe log an error to help with debugging in case it happens?